### PR TITLE
Test file generation enhancement

### LINKED ISSUE #2728: Improve sample test file creation
- [ ] Name of created file should be <currentFileName>+'_test.sol'
- [ ] Current file should be imported inside generate test file
- [ ] Define a behaviour for trying second test file for one file

It should seem like test file is created specifically for the file currently selected in file exlporer

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #2728

### CODE REVIEW

File: src/app/tabs/testTab/testTab.js [hunk 1]
```diff
@@ -8,22 +8,27 @@ class TestTabLogic {
...
 
   generateTestFile () {
-    var path = this.fileManager.currentPath()
-    var fileProvider = this.fileManager.fileProviderOf(path)
+    const path = this.fileManager.currentPath()
+    const fileName = this.fileManager.currentFile()
+    const fileProvider = this.fileManager.fileProviderOf(path)
     if (!fileProvider) return
-    helper.createNonClashingNameWithPrefix(path + '/test.sol', fileProvider, '_test', (error, newFile) => {
+    helper.createNonClashingNameWithPrefix(fileName, fileProvider, '_test', (error, newFile) => {
```
Comment (yann300) [inline src/app/tabs/testTab/testTab.js:15]: might be that no file are selected, we need to default to something, or is the button disabled in that case?

Comment (LianaHus) [reply src/app/tabs/testTab/testTab.js:15]: it is disabled

Comment (Aniket-Engg) [reply src/app/tabs/testTab/testTab.js:15]: button is disabled in that case

File: src/app/tabs/testTab/testTab.js [hunk 2]
```diff
@@ -36,12 +41,15 @@ class TestTabLogic {
     cb(null, tests, path)
   }
 
-  generateTestContractSample () {
-    return `pragma solidity >=0.4.0 <0.7.0;
+  // TODO: If currently selected file is compiled and compilation result is available,
+  // 'contractName' should be <compiledContractName> + '_testSuite'
```
[the remark this answers is not in the document]
Comment (Aniket-Engg) [reply src/app/tabs/testTab/testTab.js:45]: ok

[the remark this answers is not in the document]
Comment (Aniket-Engg) [reply src/app/tabs/testTab/testTab.js:45]: updated

**Pull request merged**